src/js/shield_defs.js

```diff
@@ -446,6 +518,13 @@ export function loadShields() {
 
   // Yukon
   shields["CA:YT"] = roundedRectShield(Color.shields.white, Color.shields.red);
+  shields["CA:YT"].overrideByRef = {
+    2: roundedRectShield(Color.shields.white, "#ce9d00"),
+    3: roundedRectShield(Color.shields.white, "#ce9d00"),
+    5: roundedRectShield(Color.shields.white, Color.shields.blue),
+    6: roundedRectShield(Color.shields.white, Color.shields.green),
+    11: roundedRectShield(Color.shields.white, Color.shields.blue),
+  };
```

Comment:
Combine with line above using `...` notation
Looking at this with fresh eyes, I think it's possible to override just the one color difference rather than repeating the whole definition.

Reply:
`textColor` is trivial to replace with `overrideByRef`, but with how it's structured, `strokeColor` would require replacing the whole `shapeBlank`. I went ahead and combined these into one declaration, but kept the helper functions in.